### DISCUSSION TITLE
Upcoming Webinar 3/16/2022

### DISCUSSION THREAD

**Discussion by shaselton-usds** [CMSgov/price-transparency-guide#397] [Announcements]
Please join the Centers for Medicare & Medicaid Services (CMS) on Wednesday, **March 16th 3:30pm – 4:30pm ET** for the fifth in a series of technical implementation webinars for the in-network negotiated rate and out-of-network historical billed charges machine-readable file requirements in the Transparency in Coverage (TiC) Final Rules (85 FR 72158). 

During this fifth webinar, the CMS team will be sure to recap January’s webinar, the release of version 1.0 of the TiC schema and introduce the TiC JSON Validator, a downloadable tool designed to help payers validate their in-network and allowed amount JSON files for errors and bugs against the TiC JSON schemas. CMS will also respond to appropriate questions that are posted in this announcement as well as in the Q&A chat box during the webinar.

Please register for the webinar [here](https://cms.zoomgov.com/webinar/register/WN_nlcqNWZHTV6yFKz8CxrAhw).

Post questions below and like/upvote questions so the team knows what's most important to you.

**Comment by tdfow** [comment]
For the optional additional_information field added to the In-Network Rate File, is there a recommended or suggested size for this string field?   Will there be an example added that contains this data element?  Also, will there be example scenarios added where the In-Network Rate file is using the CSTM-ALL scenario and/or the CSTM-00?

**Comment by tdfow** [comment]
For the scenarios where the negotiated_type is set to percentage, will there be examples posted that includes negotiated_type with 'percentage'.   Percentages generally are given in two formats, one in decimal form and then the whole number.   I believe I saw in the pull request that the expectation is to be the whole number format, such as 30%  versus as 0.30, but there is no definition on which format to use.   I feel if there isn't a specific format specified, there will be a mix of decimal and whole number formats for the negotiated_type equal to 'percentage'.

The Read.me doesn't have the distinction of which format to use, so can you provide an example or update the read.me to provide the expected percentage format for reporting?

Is the expectation to be the whole number reference?
 "negotiated_prices": [{
     "negotiated_type": "percentage",
     "negotiated_rate": 30.0,
     "expiration_date": "2022-01-01",
     "service_code": ["18", "19", "11"],
     "billing_class": "institutional"
   }]
or is it expected to be in the decimal reference?
"negotiated_prices": [{
     "negotiated_type": "percentage",
     "negotiated_rate": 0.30,
     "expiration_date": "2022-01-01",
     "service_code": ["18", "19", "11"],
     "billing_class": "institutional"
   }]

**Comment by tdfow** [comment]
There have been many schema changes since October 2021.  In addition, the community has seen many delayed responses of CMS’ guidance/feedback to the GitHub community.  Both scenarios have had impacts to the development/design/construction activities being performed for the MRFs.  For example, the percentage of charge question posted early March 2021 yet took CMS until 2/28/2022 to address (roughly a year).   Does CMS recognize the impact of these late changes/guidance being published (especially at this point in the timeline)?     Are there going to be any considerations for adjustments to the timeline due to all the delays in guidance and late breaking schema changes?

**Reply by joseph-kendall** [reply]
I would like to say that this is an important point. CMS said their changes to the schema shouldn't impact us, but in reality they do. We have an IT process to produce the files. If there are changes to the schema, it does require us to go back to our IT teams to get estimates for any changes that need to be incorporated. The IT teams then need to do any work to be ready for the 7/1 date. After development, then IT does their own testing. Finally you will have UAT and a turn process to production. All of these parts require many people and much time to complete. 

CMS and partners really should have had this in version 1.0 a year in advance of requiring the work to be done. There are still issues that need to be ironed out. If we use the new text field in the MRF, you require us to discuss how we are going to use it before doing so. This again will still add time to the process since we will have to have your OK before making a change to the process to use it. Will we be waiting a year for feedback on what we can and can't do with this field?

I appreciate the hard work on this project and ones of this size are never easy. Changes like this at the last minute aren't easy to incorporate. You may see this as having 4 months for development to be ready, but in reality our time is much less since we need time for thorough testing.

**Reply by OHCDiana** [reply]
I agree that one year would be the right amount of time to allow for all this. In a small organization, with a tiny IT staff, this is a vast amount of logic to write and test.

**Reply by RLTx1391** [reply]
Just want to echo this that the timeline (with no webinar in February and next one not until 3/16) puts pressure on implementation, particularly given a number of questions still to answer.

**Reply by shaselton-usds** [reply]
@tdfow @joseph-kendall @OHCDiana @RLTx1391 

Thanks for the honest thoughts -- really.

This has been difficult on a number of fronts, and as someone that usually operates with a technical perspective, I truly empathize with this. Especially the development lifecycle.

I think it can be sometimes lost that this isn't purely a technical project. There is a policy component that actually drives a lot of the decisions here which are downstream from it. While we try to push the technical components upstream as far as possible when having policy conversations/decisions, sometimes it is hard. This is to say that the technical and policy feedback lifecycles operate at very different speeds. What may seem like obvious and simple tech decisions (e.g. "just add 'percentage' to the `negotiated_type` and be done with it"), those decisions can't always be made without policy allowing for it. We've tried to shorten those feedback loops by putting this project on Github and using the preamble to push down a lot of the iterative technical decisions to be made here, but sometimes a change can't be made in isolation.

Our goal when initially publishing this repo, coming up on two years now, was to signal to the community and developers the core data elements that would be needed for the files and to allow time to plan for accessing required internal/external systems for that data. Every shop that is building these files is different and making assumptions on the feasibility of development with respect to timing on implementation is always going to leave people out. While we didn't know what the finalized version would look like without community feedback, we did have a good idea on the core data elements (e.g. contract data, provider data, etc) that would be needed -- this is the iterative development that has been taking place. Again, trying to not make any assumptions because each scenario is different, most of the changes over the course of the past 1.5 years haven't been requirements for data elements that would fundamentally live in different internal systems, which is where a large chunk of the complexity lives. This isn't to discount the formatting of data from these systems in containing complexity, but depending on how different IT shops are going about this, the complexity is much different. We tried to communicate that development should get started as soon as possible because the core internal systems needed access regardless of the schema layout.

From a technical perspective, the latest updates to allow for percentage of bill and per diem most likely didn't change internal development (assumptions are hard) in terms of pulling those specific contracts from a different system. Rather, what to do once that data has been pulled will need to be evaluated for those updates. We tried to keep these changes in-line with what the community was seeing in terms of the data being pulled from the internal systems to drive down the complexity on the formatting end.  Again, of course, every situation is different. It was the hope to drive down burden by keeping the internal pulled data as close as possible for reporting for these scenarios that had open questions

CMS (and the TriDepts) plans to continue to iterate on these files as new information becomes available, maybe new techniques to help further drive down the file size comes to light, etc. Policy is a bit like technology in that it isn't ever completely done, the process is always honed. By driving back the implementation timeline, it doesn't stop progress being made on these schemas. There will always be a tension between getting good changes into the schema and the timing for having those changes in production.  

We are trying to get this information to you as soon as possible while being as transparent as possible and thank you for your help in this process.

**Reply by tdfow** [reply]
@shaselton-usds appreciate the feedback and quick response.  We do understand all the hard work of both CMS and the Community to get to a compliant version for the MRF deliverables.  We do understand there is Policy involved along with their processes to identify the format of the MRFs that the mandate requires.  A critical piece for the technical and Plan community is to have the definitions for the mandate deliverable expectations, which are the published 'compliant' versions for both MRFs.
As we have seen on GitHub, the community has been making assumptions this past year while Policy was going through their process.  We understand Policy takes time to ensure the appropriate steps/expectations will address the intent of the mandate.  We even had a period of CMS going ‘dark’ on GitHub while Policy was making decisions, meanwhile the community had to make assumptions on how to address the data.  The MRFs are all about the data and the reporting of data, so clarity in understanding the data and how the data has to be organized for reporting is critical. 
Just doesn't seem realistic for the Policy process to have more time to arrive at a compliant version (3/1/2022) to identify the expectations of the community than what the Policy is requesting the technical community to deliver those MRF expectations defined on 3/1/2022.
If only the compliant versions were published on 3/1/2021 (or even when the 1st version was published in October, 2021) versus 3/1/2022, then the impact to the community for the date Policy set would be a different picture than publishing a compliant version months prior to the date Policy set.

**Reply by rajkumarsowmy** [reply]
Agree with tdfow. 
I think we all understanding and wanting balance on both sides to plan and produce MRF. 
considering any breathing room for the technical community is really appreciated! 
Because IT process has to go through layers of testing , approvals before it publish the data out to public. 
Thank you!!

**Reply by whiteck** [reply]
@shaselton-usds now that the "additional information" section has been added is this the final version for compliance or are you planning on making more changes?  My apologies if that has been clearly communicated, but, this comment above implies that maybe it's still not the final version, "CMS (and the TriDepts) plans to continue to iterate on these files as new information becomes available, maybe new techniques to help further drive down the file size comes to light, etc."

**Reply by RLTx1391** [reply]
> @shaselton-usds now that the "additional information" section has been added is this the final version for compliance or are you planning on making more changes? My apologies if that has been clearly communicated, but, this comment above implies that maybe it's still not the final version, "CMS (and the TriDepts) plans to continue to iterate on these files as new information becomes available, maybe new techniques to help further drive down the file size comes to light, etc."

I believe this is now the final compliance version (1.0.0), but additional changes may still be implemented to further optimize file sizes (but would be optional).

**Reply by joseph-kendall** [reply]
I appreciate the answers we did get in the webinar, but it seems like we will need to make extensive use of the Additional Information field to really get all of our contract details into the in-network MRFs. Where you see us as having plenty of time to implement this, we don't have enough time. We couldn't make assumptions based on something that wasn't even there in the schema until 3/1. How we handle various parts of the contract that weren't considered now have to be coded for, tested, go through UAT, and then turned to production. We have about 30 days to get everything ready for UAT. This doesn't even consider customers who want to review their plans production files before the 7/1 date.

Again as stated earlier, I do appreciate the help from CMS and the consider of the communities feedback. I just think that CMS feels like we only go in and make a simple change to the code here or there and we are done. This isn't the only compliance project we are working on nor the only project we are working on at this time. IT resources are limited not limitless. Extra work that wasn't in our original requirements documentation now has to be documented, approved, get IT estimates, approve the budget, then start the work. It is beyond aggravating that CMS doesn't seem to understand the time to make even simple changes isn't me saying to Bob, "hey make this simple change and have it done tomorrow." Most IT shops of any size don't work that way. The bigger the company the less they usually work that way in my experience.

Is there a plan to answer the questions that are in this thread? Should we expect them to be answered by a certain time?

You have a hard job. I don't envy you.

**Comment by OHCDiana** [comment]
I have heard conflicting information regarding requirements of PPO Networks as wrap networks. I have had a payor tell me that a wrap network does not need to submit an In Network Rate file, no matter how much claim volume is processed by the wrap. I've also had a payor tell me that a wrap network must submit all their rates, even if the wrap network has had little to no actual claim volume.

**Comment by eyoung490** [comment]
While I understand the Table of Contents approach from the perspective of simplifying compliance in generating machine-readable files, I think it degrades their utility and could be contrary to the intention of the original rule. We are generating MRFs for TPAs who may have a relationship with a dozen or more rented networks, which they combine in various custom ways depending on the location (regional or for each satellite office, for example) and desired network quality of their plan sponsor clients. Furthermore, some plan sponsors may bundle in direct contracted rates for a certain provider group, or a third-party that handles reference-based pricing in some situations. If a plan sponsor uses several of these networks/negotiations and provides a ToC link to each Negotiated Rate file, as required by recent clarifications, a user of that file would still know nothing about claim routing between the several networks/negotiations that may exist for the same provider and item/service. Using context clues alone, it could be possible to understand the layering of these options without further information, but in others it would be impossible. Is the intention to leave this interpretation up to the user?

**Comment by ryan-thomas-peterson** [comment]
Is the webinar a place where we can better understand the change in stance regarding usage of percentages?  I certainly understand percent of billed contracts are frequent and that it becomes much harder to determine (or estimate) a dollar amount to push to a file.  Earlier as these issues were raised, such as in discussion #23 from March of 2021, we never received guidance despite repeated requests for support in interpretation and the schema never supported this.  Users were left to assume that the tri-departments would not comment or support a percent of billed method since it was raised, asked about, and never incorporated.  This made some sense, since the rules stated dollar amounts should be provided.  

The concern on this somewhat dramatic new support is that if the intent of these files is to layer toward consumer shoppable tools and innovators to use this data to support comparability and transparency, doesn't that suddenly put the onus on them (who are much further from the contracts) to determine or estimate the dollar amount in some other way?  Don't we lose that comparability?

This new support likely comes with a sigh of relief for this community (the ones building the files) because it can align with their contract and they don't need to derive or estimate a dollar amount.  But, this change is likely to promote even more requests to try to further allow the schema to 'fit' all the complex terms inherent in so many existing contracts that push further away from comparability  (what about DRG outlier payments based on Length Of Stay?  What about AWP drug pricing rendered on the medical side?  What about anesthesia services where we contract per 15 minute interval?  What about zip code specific or patient demographic specific pricing?  etc. etc).  Until this inclusion the assumption was that the interpretation to 'fit' existing complex contracts into this existing schema allocated the responsibility to the producer of these files to best interpret their contracts and fit the contract terms to the schema to support future comparability and downstream efforts.

If it was planned to be supported, why was this not communicated before to allow for discussion among the technical community about implications?   Dropping in support the day before locking in a V1.0.0 schema feels like a major decision with far reaching downstream implications for the usability of these data that warrants more explanation.

**Comment by RLTx1391** [comment]
Is there any guidance on data retention periods? https://github.com/CMSgov/price-transparency-guide/discussions/238

Are there any ways to protect against potentially bad actors downloading the same file multiple times, spiking egress charges?
https://github.com/CMSgov/price-transparency-guide/discussions/180

**Comment by joseph-kendall** [comment]
We also need an answer to this question before moving ahead with publishing files. [https://github.com/CMSgov/price-transparency-guide/discussions/356](url)

**Comment by lkabi** [comment]
If a Plan (with plan_id_type as  'EIN') has 'plan_market_type' as both 'group' **and** 'individual' , can we populate 'plan_market_type' as **'both'** ? please give us guideline to incorporate both scenarios.

**Reply by lkabi** [reply]
@shaselton-usds - We have Plans asking us how to handle if a network has both Individual and Group (plan_market_type). Please give a direction on how to handle it

**Reply by shaselton-usds** [reply]
I'm guessing even though the EIN is the same, the product ID (HIOS) is different thus the different market types, correct? Is there a reason why you couldn't use a table of contents file to reference the same in-network file?

```json
{
   "reporting_entity_name": "abc-corp",
   "reporting_entity_type": "abc-corp",
   "reporting_structure": [{
     "reporting_plans": [{
       "plan_name": "plan A",
       "plan_id_type": "EIN",
       "plan_id": "11111111111",
       "plan_market_type": "individual"
     },{
       "plan_name": "plan A",
       "plan_id_type": "EIN",
       "plan_id": "11111111111",
       "plan_market_type": "group"
     }],
     "in_network_files": [{
       "description": "in-network file",
       "location": "https://www.some_site.com/files/in-network-file-123456.json"
     }]
  }]
}
```

**Comment by NMerrick58** [comment]
I have recently put the following question within the Discussion post of Github, I was hoping to receive information from CMS regarding this as we do not think it is flesched out within rule. Thanks!

Can you please get into how plans are to post leased networks Machine Readable Files? Are these files needed to be posted as the leased network name aside from the regional network file, or are the regional Issuer supposed to aggregate the information within their MRF files of their information?

**Comment by RLTx1391** [comment]
Could we discuss the scenario where rates may vary based on location such as zip code? I know this came up in #189.

**Comment by Stephanie-Raymond** [comment]
Re-submitting questions that are currently logged as outstanding issues:

**Q #280: "Description" Data Element for In-Network Negotiated Rates**
For information being reported on the in-network negotiated rates for all items and services, specifically the “Description” data element located under the In-Network Object, please confirm whether this field is indeed 'Optional' as currently defined in the technical guidance or does CMS expect it to be populated whenever information is available in IT systems/database?
Similarly, is it correct to interpret that the 'Billing Code’ under the In-Network Object is intended to mean the Billing Code Description' for this field?
Image: https://user-images.githubusercontent.com/94474475/142049178-a1e70d90-9982-4054-9775-ada9cdd231ad.png

**Q #281: In-Network Object Fields: Billing Code Elements**
Is it correct to interpret that, if the 4 fields below are populated (appearing under the "In-Network" object), then the same set of these fields appearing under the 'Bundle Code object' and 'Covered Services Object" can be left 'blank'?
If interpretation above is not true/correct, can CMS/GitHub provide guidance/illustration on what cases can co-exist.
If interpretation above is true/correct, can Plans further interpret that for every 'Provider rate record', only 1 out of the 3 sets (of these 4 fields) would be populated while the other 2 would be 'blank'?
Image: https://user-images.githubusercontent.com/94474475/142050063-8a321c9a-4246-4bc1-b327-72906cdaa899.png

**Q #282 Bundle Example for JSON Display for DRG Claim**
There are several issues we find with DRG pricing, and how to fit into the allowed format.
1.	How is severity listed? Example: DRG 001 has severity options of 1 and 2. Should this be listed as 001.1 and 001.2?
2.	Can DRG payments be displayed as FFS type with the DRG billing type as the corresponding rate? If using bundle, listing encounter header information, every Diagnosis (ICD) code and Surgical Code or encounter detail line information, revenue code of each detailed line, that COULD be in that DRG (as well as many others) is something that is generally standard.  Can CMS provide an illustrative example of how DRG must be bundled?

**Q #284 Technical Guidance on Capitation for In-Network File**
Can CMS/GitHub provide additional guidance and clarifications about technical specifications on how a Plan should handle capitation for purposes of the In-Network Machine Readable File?
Specifically, is the required reportable information the financial arrangement of per member/per month/quarter between the provider and insurer?
Is the plan instructed to provide this capitation rate in lieu of fee for service for associated procedure codes? Or, can capitated services be reported as “encounter” services (the amount the encounter for service would be paid if FFS).

**Comment by Stephanie-Raymond** [comment]
Can CMS clarify the **retention policy** for Plans maintaining machine readable files? Would it be reasonable to keep only current month of MRF files on one the public site and archive previous month MRF files from the public site then make them available upon request for 12 months?

**Reply by nhartmanNWA** [reply]
Making them available upon request seems excessive depending on how the website is set up to provide access to the files. The files are refreshed every 30 days which provides ample opportunity for an entity to download the files. Additionally, storing 12 months of files of the INN size would require significant storage.

**Comment by Stephanie-Raymond** [comment]
Can CMS **clarify the schema definition (JSON file)**? What is provided does not match the schema description on GitHub site.

**Reply by shaselton-usds** [reply]
@Stephanie-Raymond We hope that the definitions match the documentation -- can you provide some links to where this might not be the case?

**Comment by klomurray10** [comment]
Does CMS have a plan for additional versioning and changes to the MRF schemas? Will CMS be publishing on a set cadence going forward or be dropping version updates in the manner we saw so far where new schema updates are announced as they are dropped?

**Reply by shaselton-usds** [reply]
Iterative development is planned to continue.

**Comment by whiteck** [comment]
Can we utilize the external references within each MRF or can it only be utilized in the table of contents structure?
- EX: if we have Behavioral rates that stay the same across multiple plans/products, can we just link to a separate rate file rather than reproducing that data on every file.  This would be utilized in a single plan file.

**Reply by shaselton-usds** [reply]
External in-network references can only be made in the table of contents. 

External provider networks can be made inside the in-network file themselves.

**Comment by AJLgeHG** [comment]
The topic of this question is "Which Entity to Report 'Plan Name' - Network vs Health Plan Issuer"
It is Item #388 on Discussions

Plan name is required for both the In-Network (INN) file and the Table of Contents (TOC) file. We contract with different networks to provide coverage for our members. As such, these networks will provide us the INN MRF and we will create a TOC file which will have information for the INN MRFs. The networks will not provide Plan_id_type (HIOS/EIN), plan_id (HIOS/EIN value), and plan_market_type. We have this information and will provide it in the TOC file.

On our end the HIOS and EINs are aligned with Plan names that are not the same as those from the network. The network Plan Names are the labels they assign to their plan offerings to our members and our Plan Names are tied to the payer entity/employer groups with whom we contract to provide coverage for our members. For both the networks and us the plan names are different but correct for each of us.

So, the question is, which Plan Name should be used for the INN MRF? Should it be ours (the payer/employer groups) or the network's (their designated name for the plans they offer to members)?

**Reply by AJLgeHG** [reply]
Hi @shaselton-usds - Reaching out on this question as you'd suggested on the Webinar

**Reply by shaselton-usds** [reply]
Thanks for the follow up @AJLgeHG 

> Plan name is required for both the In-Network (INN) file and the Table of Contents (TOC) file. 

Take a look at the documentation [here](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#additional-notes-concerning-plan_name-plan_id_type-plan_id-plan_market_type). In short, you don't need the `plan_name`, `plan_id_type`, `plan_id`, `plan_market_type` in the actual in-network file if you plan to use a table of contents. Does that clear up your issue?

**Reply by AJLgeHG** [reply]
Thanks, @shaselton-usds. In the example provided could you respond to the question of which Plan Name should be used for the INN MRF? Should it be ours (the payer/employer groups) or the network's (their designated name for the plans they offer to members)? 

Or, does it not matter whether it's the payer or the the network's? Currently, we're planning on having it be the payer. Please confirm if that would be correct.

**Reply by RLTx1391** [reply]
@shaselton-usds is there an answer to @AJLgeHG's question?

**Comment by joseph-kendall** [comment]
Will you be discussing how we can use the additional information field that was added to version 1.0 of the schema?

What scenarios should we use for bundling? Since we don't have a claim going through our contract, do we need to report such things as multiple surgeries on a claim that would only pay 1 of the surgeries (this sounds like what you mean by bundling)? There could be thousands or millions of these permutations. Could you please clarify with some scenarios on what you mean by bundling?

Again since we don't have a claim running through our contract, how do we report things that depend on a claim? Scenarios might be number of days, line charges, lesser of language, stop loss, primary diagnosis code, etc.

With version 1.0 of the schema, I feel we still have many holes in the schema. There are several scenarios that aren't captured by the schema. These scenarios aren't specific to one plan/issuer, but they are industry wide. Without a claim going through the contract, do we only report what we can from the contract and ignore the pieces of the contract that rely on a claim? How do we handle these areas in the schema where it doesn't tell us how to get the contract details from our contracts?

**Comment by cfd822** [comment]
If a plan has multiple in-network rate files (for example, an in-network rate file for its medical services from its medical TPA and a second in-network rate file for its behavioral health services from a separate TPA), is the plan required to aggregate the in-network rate files and create a Table of Contents file or can the plan post multiple in-network rate files from different TPAs?  Based on the Technical Clarifications Q&A, it appears the aggregation and Table of Contents would be optional.

**Reply by shaselton-usds** [reply]
If you plan to leverage multiple external in-network files to define a complete in-network file for your plan, then you would need to use a table of contents to point to all of the in-network files. 

Both leveraging external in-network files and using a table of contents are optional, but if you choose to use external in-network files, then the table of contents will be required.

**Comment by tdfow** [comment]
Where can we find the validator information shared in the CMS webinar today?

**Reply by lkabi** [reply]
Here's the link - https://github.com/CMSgov/price-transparency-guide-validator

**Reply by shaselton-usds** [reply]
Thanks @lkabi! I'll update the README.md here to point to it.

**Reply by Rashmi-sb** [reply]
Has anyone used this validator, we tested it and we see some of required the JSON entities names are not validated  against correct schema  entities name 
Eg : Billing_code_type(schema naming ) is not validated to BillingCodeType(incorrect naming)  , even if we remove this whole element from the output JSON  , the validator says Valid JSON .

**Reply by shaselton-usds** [reply]
@Rashmi-sb Would you mind either opening up an [issue](https://github.com/CMSgov/price-transparency-guide-validator/issues) or [discussion](https://github.com/CMSgov/price-transparency-guide-validator/discussions) on that repo if you're running into issues?

**Reply by Rashmi-sb** [reply]
@shaselton-usds thanks , Started discussion as well added an issue.

**Comment by Data-san** [comment]
Can CMS please post the webinar transcript from March 16th. Thanks much.

**Reply by shaselton-usds** [reply]
It is now posted [here](https://www.cms.gov/healthplan-price-transparency/resources/webinars). Let me know if you run into any issue accessing it.

**Reply by kayroQ** [reply]
I don't see the webinar link posted (such as for January webinar), just the link to register for the webinar.

**Reply by shaselton-usds** [reply]
Maybe it is a caching issue? Try clearing out your cache (or opening the link in private/incognito mode) and let me know.

**Reply by Data-san** [reply]
Yes, incognito mode worked. I see the transcript. Why is that? Thank you!

**Reply by shaselton-usds** [reply]
@Data-san your browser is caching an old version of the site, you need to clear that out (or the browser will clear it out with a new cache eventually).

**Reply by olderthandirt5** [reply]
@shaselton-usds  The link for the 3/16 webinar is the same as the link to the 11/30/2022 webinar. Even the passcode listed is the same. This is as of 6:30 PM Central

**Reply by olderthandirt5** [reply]
The link is still pointing to the 11/30/2021 webinar as of 6:30 AM 3/24/2022

**Reply by tdfow** [reply]
![image](https://user-images.githubusercontent.com/67592775/160035423-7fe00acb-4608-4506-b470-265dc605afa3.png)

**Reply by tdfow** [reply]
The link to the webinar is not the 3/16/2022 Webinar.  It is the link to the earlier discussion.

**Reply by bettinavanover** [reply]
Can we please get the updated link?  The link is pointing us to the November webinar.

**Reply by shaselton-usds** [reply]
Thanks for pointing that out -- link has been updated. Let me know if you are still finding issues.

**Reply by olderthandirt5** [reply]
Thanks!! It is all good now.

**Comment by OHCDiana** [comment]
I was able to successfully validate an In Networks Rates file using the validator. My test file had "billing_code_type": "CPT"

**************************************
Diana Jacobs
Senior Programmer / Analyst
Ohio Health Choice
330-996-8205  /  800-554-0027
***@***.******@***.***>
**************************************

**Comment by RLTx1391** [comment]
Will there be another webinar this month? It looks like folks still have a number of open questions, based on looking at the discussions on here and it may be of value to go through and answer these.
